src/main/java/edu/ucsb/cs156/happiercows/controllers/AnnouncementsController.java

```diff
@@ -67,10 +73,26 @@ public ResponseEntity<Object> createAnnouncement(
             }
         }
 
-        if (startDate == null) { 
+        // Fix timezone difference for PST
+        if (startDate != null) {
+            Calendar calendar = Calendar.getInstance(); 
+            calendar.setTime(startDate);
+
+            calendar.set(Calendar.HOUR_OF_DAY, 8); 
+            startDate = calendar.getTime();
+        }
+        else {
             log.info("Start date not specified. Defaulting to current date.");
             startDate = new Date(); 
         }
+        if (endDate != null) {
+            Calendar calendar = Calendar.getInstance(); 
+            calendar.setTime(endDate);
+
+            calendar.set(Calendar.HOUR_OF_DAY, 8); 
+            endDate = calendar.getTime();
+        }
+
```

Comment:
There's a better way to do this (actually I can think of several better ways to do this.)

But we'll leave it to the S25 students to fix it.